frontend/src/app/modules/auth/auth-routing.module.ts

```diff
@@ -1,18 +1,24 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginPageComponent } from './login-page/login-page.component';
 
 import { RegisterPageComponent } from './register-page/register-page.component';
 
 
 const routes: Routes = [
     {
-        path: '',
-        component: RegisterPageComponent
+        path : 'register', component : RegisterPageComponent
+    },
+    {
+        path : 'login', component : LoginPageComponent
+    },
+    {
+        path : '', redirectTo: 'register', pathMatch: 'full'
```

Comment:
Redirect to login
And match should be none (So, /ajsdgh will be also redirected to login